pkgs/development/libraries/openssl/default.nix

```diff
@@ -205,4 +205,16 @@ in {
     ];
     withDocs = true;
   };
+
+  openssl_3_0 = common {
+    version = "3.0.0";
+    sha256 = "sha256-We7fy0bCUhTJvTftYHgpe03wHQEiZ/6enu4x9hvHBTY=";
+    patches = [
+      ./3.0/nix-ssl-cert-file.patch
+      (if stdenv.hostPlatform.isDarwin
+       then ./1.1/use-etc-ssl-certs-darwin.patch
```

Comment:
Shouldn't these patches come from the 3.0 folder or are they the same and the ones in this PR aren't required anymore?

Reply:
They're the same, I just copied them and forgot to update the reference.